code/modules/research/designs.dm

```diff
@@ -604,7 +604,45 @@
 	build_path = /obj/item/weapon/gun/energy/cutter/plasma
 	sort_string = "TAEAC"
 
-
+/datum/design/item/weapon/divet
+	name = "Winchester NK Divet pistol"
+	id = "divet"
+	req_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 5)
```

Comment:
You can make the tech requirements 1 and 3 respectively. Divet is a powerful gun, yes, but not complex in design.